utility/util.py

```diff
@@ -212,9 +213,8 @@ def prepare_to_indexing(yc_api_prefix: str, modules_to_index, credentials: list,
                                                          module['name'],
                                                          module['revision'],
                                                          module['organization'])
-                response = requests.get(url, auth=(credentials[0], credentials[1]),
-                                        headers={'Content-Type': 'application/json',
-                                                 'Accept': 'application/json'})
+                response = requests.get(url, headers={'Content-Type': 'application/json',
```

Comment:
create json_headers and replace only in scripts that you are changing in this patch

Reply:
done - separate PR will be created for replacing this in rest of the scripts